code-challenges.test.js

```diff
@@ -23,45 +23,132 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+
+describe("divisibleBy3", () => {
+    it("checks to see if a number is divisible by 3", () => {
+        const object1 = { number: 15 }
+        // Expected output: "15 is divisible by three"
+        const object2 = { number: 0 }
+        // Expected output: "0 is divisible by three"
+        const object3 = { number: -7 }
+        // Expected output: "-7 is not divisible by three"
+        expect(divisibleBy3(object1)).toEqual("15 is divisible by three")
+        expect(divisibleBy3(object2)).toEqual("0 is divisible by three")
+        expect(divisibleBy3(object3)).toEqual("-7 is not divisible by three")
+    })
+})
 
 // b) Create the function that makes the test pass.
+const divisibleBy3 = (object) => {
+    let myNot = "not "
+    if (object.number % 3 === 0){
+        myNot = ""
+    }
+    return `${object.number} is ${myNot}divisible by three`
+
+}
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
+// Pseudo code: Create a function called divisibleBy3 that takes in a single parameter
+// The parameter is an object, and inside of the object, is the number to test for division by 3
+// Use the modulo operator to determine if the number evenly divisible by 3
+// Use conditionals and string interpolation to return the desired output
+
+
 
-// Pseudo code:
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
+
+
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+describe("arrayCapitalizer", () => {
+    it("capitalizes all the words in an array", () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+        // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+        // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+        expect(arrayCapitalizer(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(arrayCapitalizer(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+
+    })
+})
 
 // b) Create the function that makes the test pass.
+const arrayCapitalizer = (array) =>  array.map(value => value.split('').map((value, index) => {
```

Comment:
Great logic here. I recommend actually opening up the code blocks for each map and having everything on separate lines to make it much easier to read, as having some things on one line with chaining and then opening up one thing on multiple lines makes it look look like you did not put effort in to make it legible.